src/components/paragraph/rows/rows.tsx

```diff
@@ -1,37 +1,54 @@
 import OneColumn from "@/components/paragraph/rows/one-column";
 import TwoColumn from "@/components/paragraph/rows/two-column";
 import ThreeColumn from "@/components/paragraph/rows/three-column";
-import {LayoutParagraph, StanfordParagraph} from "@/lib/drupal/drupal";
+import {ParagraphLayout, ParagraphUnion} from "@/lib/gql/__generated__/drupal";
+import {getParagraphBehaviors} from "@/components/paragraph";
 
-interface RowProps {
-  items: StanfordParagraph[]
+type RowProps = {
+  items: ParagraphUnion[]
   fullWidth?: boolean
 }
 
-export const ParagraphRows = ({items, fullWidth = true, ...props}: RowProps) => {
-  const layouts: Record<string,  {layout: LayoutParagraph, children: StanfordParagraph[]}> = {};
+type Layout = Record<string, {
+  item: ParagraphLayout
+  layout: string
+  config?: Record<string, any>
+  children: ParagraphUnion[]
+}>
+
+export const ParagraphRows = ({items, fullWidth}: RowProps) => {
+  const layouts: Layout = {};
 
   // Set the layouts first.
   items.map(item => {
-    if (item.type === 'paragraph--layout' && item.behavior_settings.layout_paragraphs.layout) {
-      layouts[item.id] = {layout: item, children: []}
+    if (item.__typename === 'ParagraphLayout') {
+      const behaviors = getParagraphBehaviors(item);
+
+      layouts[item.id] = {
+        item,
+        layout: behaviors.layout_paragraphs?.layout || 'sul_helper_1_column',
+        config: behaviors.layout_paragraphs?.config,
+        children: []
+      }
     }
   })
 
   // Add the components to each of the layouts.
   items.map(item => {
-    const parentUUID = item?.behavior_settings?.layout_paragraphs?.parent_uuid;
+    const behaviors = getParagraphBehaviors(item);
+    const parentUUID = behaviors?.layout_paragraphs?.parent_uuid
     if (parentUUID && layouts[parentUUID]) {
       layouts[parentUUID].children.push(item);
     }
   })
 
   return (
-    <div className="grid grid-column mb-[40px] gap-[90px]" {...props}>
+    <div className="grid gap-32 mb-10">
```

Comment:
@pookmish Here's the margin change I was referring to.

Reply:
huh. yeah definitely don't remember that one. Oh well... like you said, they probably would prefer this.